envs/py36/requirements.txt

```diff
@@ -65,7 +65,7 @@ prometheus-client==0.10.1
 prompt-toolkit==3.0.18
 ptyprocess==0.7.0
 pycparser==2.20
-PyGEL3D==0.0.27
+PyGEL3D==0.1.4
```

Comment:
This reverts commit 921ce717, i.e. we can remove toggling the version from the PR history.